lib/license.js

```diff
@@ -153,3 +220,28 @@ function writeTemplatedLicense(newName, oldName, text, pkgName) {
   } catch(_) {
   }
 }
+
+function hasOthers() {
+  var old = readLicense();
+  return old && /strongloop.*other.*contributors/i.test(old);
```

Comment:
The check for `null`/`false`/`''` can be skipped here. That's the beauty of using `RegExp.test()` instead of `String.match()`, you can pass in just about anything :-)
```js
return /strongloop.*other.*contributors/i.test(readLicense());
```